packages/tokens/src/global/elevations.android.ts

```diff
@@ -0,0 +1,13 @@
+export interface ElevationsProps extends Array<number> {
```

Comment:
Same here, regarding the @MicaelRodrigues' optional suggestion
```suggestion
export interface ElevationsProps extends number[] {
```